SPECS/dracut/dracut.spec

```diff
@@ -156,6 +156,7 @@ ln -srv %{buildroot}%{_bindir}/%{name} %{buildroot}%{_sbindir}/%{name}
 %dir %{dracutlibdir}/modules.d
 %{dracutlibdir}/modules.d/*
 %exclude %{_libdir}/kernel
+%exclude %{_libdir}/dracut/modules.d/20overlayfs/*
```

Comment:
Does this exclude the `20overlayfs`? Or does installing the package leave an empty directory? Should this be `%exclude %{_libdir}/dracut/modules.d/20overlayfs` instead?

Reply:
Fixed, thanks.